test cases/java/9 jdk/meson.build

```diff
@@ -0,0 +1,10 @@
+project('jdkjava', ['c', 'java'])
+
+fs = import('fs')
+
+java = find_program('java', required: true)
```

Comment:
Not really sure what to about this since the Java executable that will be found won't necessarily be the one in JAVA_HOME/bin

Reply:
I think that this is best served by being fixed in a java module which I have been working on